infra/api/app-config/env-config/scheduled_jobs.tf

```diff
@@ -0,0 +1,51 @@
+locals {
+  # The `task_command` is what you want your scheduled job to run, for example: ["poetry", "run", "flask"].
+  # Schedule expression defines the frequency at which the job should run.
+  # The syntax for `schedule_expression` is explained in the following documentation:
+  # https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-scheduled-rule-pattern.html
+
+  # See api/src/data_migration/command/load_transform.py for argument specifications.
+  load-transform-args = {
+    # Runs, but with everything disabled.
+    dev = [
+      "poetry",
+      "run",
+      "flask",
+      "data-migration",
+      "load-transform",
+      "--no-load",
+      "--no-transform",
+      "--no-set-current",
+    ],
+    staging = [
+      "poetry",
+      "run",
+      "flask",
+      "data-migration",
+      "load-transform",
+      "--load",
+      "--transform",
+      "--set-current",
+    ],
+    prod = [
+      "poetry",
+      "run",
+      "flask",
+      "data-migration",
+      "load-transform",
+      "--load",
+      "--transform",
+      "--set-current",
+    ],
+  }
+  scheduled_jobs = {
+    copy-oracle-data = {
+      task_command        = ["poetry", "run", "flask", "data-migration", "copy-oracle-data"]
+      schedule_expression = "rate(2 minutes)"
```

Comment:
Should we include enabled/disabled here too, or does a null schedule or something like that cause the job to just not run? Just thinking there may be times we want to keep the job from running, but not delete it from here entirely.

Reply:
Added! https://github.com/HHS/simpler-grants-gov/pull/2506/commits/1ba3d0615fdff173d9020b34a7e7fc83cfe206e9